release/test_release_gold.sh

```diff
@@ -39,6 +50,13 @@ function tear_down {
 	unset _PROJECTS_DIR
 }
 
+function test_check_usage {
+	local check_usage_output=$(check_usage)
+	local expected_check_usage_output=$(cat test-dependencies/expected/check_usage_output.txt)
+
+	assert_equals "${check_usage_output}" "${expected_check_usage_output}"
```

Comment:
Good job, @lucasmiranda0!

Let's save the variables declarations since they're used just once, please see:
```
assert_equals \
    "$(check_usage)" \
    "$(cat test-dependencies/expected/check_usage_output.txt)" \
```